en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-websocket-api-in-choreo-connect.md

```diff
@@ -1,43 +1,108 @@
+You can deploy a WebSocket API in the following ways depending on the Choreo Connect **mode** you have chosen.
 
-## Choreo Connect with WSO2 API Manager as a Control Plane
 
-Follow the instructions below to use Choreo Connect with WSO2 API Manager as the Control Plane to deploy a WebSocket type Streaming API via the Publisher Portal in WSO2 API Manager:
+|**Mode**         | **Method**    |
+|--------------|-----------|
+|[Choreo Connect with WSO2 API Manager as a Control Plane]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/apim-as-control-plane/)   | [Via WSO2 API Manager Publisher Portal](#via-wso2-api-manager-publisher-portal)  |
+|[Choreo Connect as a Standalone Gateway]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/concepts/as-a-standalone-gateway/)  |[Via apictl for Standalone Mode](#via-apictl-for-standalone-mode) |
 
-!!! info
-    **Before you begin**
 
-    This guide assumes that you already have a Choreo Connect instance that is up and running. If not, checkout the [Quick Start Guide]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide-docker-with-apim/) on how to install and run Choreo Connect. To learn more about Choreo Connect, have a look at the [Overview of Choreo Connect]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview).
+!!! tip "Characteristics of WebSocket APIs Deployed in Choreo Connect"
+    - The API is created based on the **Channels** defined in the AsyncAPI definition. They are also known as **Topics** in WSO2 API Manager, and have a slight similarity to **Resources** in a REST API.
+    - Once the API is exposed, although the Gateway endpoint includes a basepath (with topic paths if defined), the **Connection Request** will be sent to the original backend endpoint you provided without any appended paths.
+    - If you have a requirement to append different **paths** to the backend URL for different **Channels** (**Topics**), a URL-mapping can be added. The **Topics** that do not include a URL-mapping will have the usual behavior as mentioned above.
+    - WebSocket APIs support OAuth 2.0 Security and each **Topic** can have its own local scope.         
+    - If configured, Choreo Connect is also capable of publishing analytics for WebSocket frames sent and received by the client.         
+    - Currently, the properties such as URL Mapping and Security are supported at the topic action level with a similarity to REST APIs. Therefore, in Websocket APIs these must be defined under **Publish** or **Subscribe**. Yet, from a Gateway point of view, since both Publish and Subscribe are initiated with a GET request that appears the same, if a **Topic** has properties defined under both Publish and Subscribe, the properties for **Publish** will be prioritized.
```

Comment:
```suggestion
    - Currently, the properties such as URL Mapping and Security are supported at the topic action level with similarity to REST APIs. Therefore, in WebSocket APIs, these must be defined under **Publish** or **Subscribe**. Yet, from a Gateway point of view, since both Publish and Subscribe are initiated with a GET request that appears the same, if a **Topic** has properties defined under both Publish and Subscribe, the properties for **Publish** will be prioritized.
```